src/services/dashboards/DashboardsLNB.vue

```diff
@@ -128,9 +89,81 @@ export default defineComponent({
             ]),
         });
 
-        const filterFavoriteItems = (menuItems: LNBItem[] = []) => {
+        const mashUpProjectGroup = (dashboardList: ProjectDashboardItem[] = []): LNBMenu[] => {
+            const noGroupList = [] as ProjectDashboardItem[];
+            const groupProjectDashboardMap = {} as Record<string, ProjectDashboardItem[]>;
+            // Since all lnbitem do not have groups, and even if they do have groups, they are not sorted.
+            // Items with groups are converted to map format, and if there are no groups, they are put in noGroupList.
+            dashboardList.forEach((d) => {
+                // With group
+                if (state.projectItems[d.project_id]) {
+                    if (groupProjectDashboardMap[state.projectItems[d.project_id].data.groupInfo.name]) {
+                        groupProjectDashboardMap[state.projectItems[d.project_id].data.groupInfo.name].push(d);
+                    } else {
+                        groupProjectDashboardMap[state.projectItems[d.project_id].data.groupInfo.name] = [d];
+                    }
+                }
+
+                // No-group
+                noGroupList.push(d);
+            });
+
+            // Result to return
+            const result = [] as LNBMenu[];
+
+            // The mapped group items are in the form of an array along with the title, and each item is mapped to LNBItem type and pushed to result.
+            Object.keys(groupProjectDashboardMap).forEach((d) => {
+                result.push([
+                    {
+                        type: MENU_ITEM_TYPE.TITLE,
+                        label: d,
+                        id: d,
+                        foldable: true,
+                    },
+                    ...groupProjectDashboardMap[d].map((board) => ({
+                        type: MENU_ITEM_TYPE.ITEM,
+                        id: board.project_dashboard_id,
+                        label: board.name,
+                        to: {
+                            name: DASHBOARDS_ROUTE.DETAIL._NAME,
+                            params: {
+                                dashboardId: board.project_dashboard_id,
+                            },
+                        },
+                        favoriteType: FAVORITE_TYPE.DASHBOARD,
+                    })),
+                ]);
+            });
```

Comment:
suggestion: It would be nice to have a sort by project group & board name. 
The order of `Object.keys(groupProjectDashboardMap)` seems to change every time.